code/game/objects/items.dm

```diff
@@ -664,6 +664,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		A.Remove(user)
 	if(item_flags & DROPDEL)
 		qdel(src)
+	item_flags &= ~BEING_REMOVED
```

Comment:
What does  it change?

Reply:
When you attempt to break out of handcuffs, this flag acts as a debounce, being set to true on the proc on the carbon that handles resisting cuffs, and is set to false when the the proc ends, which includes a do after. Because this PR makes it so cuffs can be dropped while this resist cuff proc is still going on, it's possible for these cuffs to be use on another person while this debounce is still set to true, resulting in a bug where the new cuffed person cannot start resisting out of the cuffs until the old person with the cuffs finished resisting out of the cuffs that are already off.